src/init.ts

```diff
@@ -281,8 +281,10 @@ export async function init(options: Options): Promise<boolean> {
   if (!options.dryRun) {
     // --ignore-scripts so that compilation doesn't happen because there's no
     // source files yet.
+    let command = getPkgManagerName(options.yarn);
+    if (process.platform === 'win32') command += '.cmd';
```

Comment:
I'd suggest to untangle it a little bit, and make it as API for `getPkgManagerName`, like:

```
getPkgManagerName(options.yarn, process.platform)
```

This logic (with `cmd` or without one) would be encapsulated and also easily covered with unit tests. What do you think?